src/libcollectionstest/str.rs

```diff
@@ -468,6 +468,18 @@ fn test_is_utf8() {
     assert!(from_utf8(&[0xEF, 0xBF, 0xBF]).is_ok());
     assert!(from_utf8(&[0xF0, 0x90, 0x80, 0x80]).is_ok());
     assert!(from_utf8(&[0xF4, 0x8F, 0xBF, 0xBF]).is_ok());
+
+    // deny embedded in long stretches of ascii
```

Comment:
Entirely reasonable, no reason to share test name there, no common setup or anything. Fixed to have its own test function.